src/Tizen.Multimedia.MediaPlayer/Player/Player.Properties.cs

```diff
@@ -667,6 +676,57 @@ public float AudioPitch
             }
         }
 
+        /// <summary>
+        /// Gets or sets the default codec type of the audio decoder.
+        /// </summary>
+        /// <value>A <see cref="CodecType"/> that specifies the type.
+        /// The default codec type could be different depending on the device capability.</value>
+        /// <remarks>
+        /// <para>To set, the player must be in the <see cref="PlayerState.Idle"/> state.</para>
+        /// <para><see cref="CodecType.Sw"/> is supported basically.
+        /// If H/W audio codec type is not supported in some cases, S/W audio codec type could be used instead.</para>
+        /// <para>The availability could be changed depending on the codec capability.
+        /// If an application wants to use the H/W audio codec type as default,
+        /// following functions should be called after the codec type is set. :<br/>
+        /// <see cref="AudioEffect.IsAvailable"/><br/>
+        /// <see cref="EnableExportingAudioData"/><br/>
+        /// <see cref="DisableExportingAudioData"/><br/>
```

Comment:
In the above description, `following functions should be called after the codec type is set.` is described.
If you describe `EnableExportingAudioData` and `DisableExportingAudioData` together here,
user can confuse whether it should be called the at the same time or not.

Reply:
i dont think so. the meaning of enable and disable is clear.